### ISSUE TITLE
Websocket Support

### ISSUE THREAD

**Issue by bcomerford** [stripe/stripe-apps#102]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
It's slightly confusing to determine the validity of my custom CSP rules and whether or not the `[Report Only]` logs are actually restrictive or not, but I'm attempting to initiate a `wss` connection and the error is hard to parse yet still appears that the CSP policy is not allowing the initial request.

**Describe alternatives you've considered**
I've added `wss://` rules to the policy but haven't had success. I think these would be important to establish customer messaging or other real-time like updates to the UI.

**Comment by hunterclarke** [comment]
I use Supabase for the backend of my app. One of the benefits of using Supabase, Firebase, etc. is the ability to watch queries for updates. 

It wouldn't be uncommon for two separate users of the same Stripe account to be viewing the same form or data in my app. So if one decides to take action, allowing the other to see that change immediately instead of on refresh/navigation would be a better user experience.
